portfolio/src/main/webapp/contact.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/e073fdea10.js" crossorigin="anonymous"></script>
+    <script src="contact-switch.js"></script>
+  </head>
+  <body>
+    <div class="contact-container">
+      <div class="navbar">
+        <a href="contact.html">contact</a>
+        <a href="projects.html">projects</a>
+        <a href="index.html">about</a>
+      </div>
+      <div class="usernames-container">
+        <div class="business-tab" onclick="personalToBusiness()">BUSINESS</div>
+          <div id="business-body" class="business-body">
+            <div class="account-container">
+              <a class="account-icon" href="https://www.linkedin.com/in/ashley-granitto"><i class="fab fa-linkedin-in"></i></a>
```

Comment:
If you'd like to practice more javascript, you could probably create a function to append these elements. Since the template seems to be the same, you can iterate over an array of accounts with {username, icon} and append them onto the #business-body.

Reply:
This is a great idea--thank you! I added two functions to do the business and the personal bodies (as they have opposite structure)